apps/notifier/src/pull-listener.ts

```diff
@@ -1,7 +1,13 @@
 import { logger, PubSubClient } from '@bike4life/commons'
 import { Message } from '@google-cloud/pubsub'
+import  sgMail = require('@sendgrid/mail')
+import fs = require("fs");
+import nunjucks = require("nunjucks");
+import path = require('path');
+import { toUSVString } from 'util';
 
-const { PUBSUB_PROJECT_ID, NOTIFIER_PUBSUB_SUBSCRIPTION_NAME, NOTIFIER_PUBSUB_TOPIC_NAME } = process.env
+
+const { PUBSUB_PROJECT_ID, NOTIFIER_PUBSUB_SUBSCRIPTION_NAME, NOTIFIER_PUBSUB_TOPIC_NAME, TO_REPLY_TO_EMAIL_SENDGRID, SENDGRID_APY_KEY } = process.env
```

Comment:
That's not the proper place to use environment variables. There is a file called `settings.ts` where we're managing the environment variables. We should export a new object with the required configuration for sendgrid called `sendgridSettings` or something like that